openfl-workspace/workspace/plan/defaults/network.yaml

```diff
@@ -7,4 +7,5 @@ settings:
     client_reconnect_interval  : 5
     require_client_auth        : True
     cert_folder                : cert
-    enable_atomic_connections  : False
\ No newline at end of file
+    enable_atomic_connections  : False
+    enable_remote_attestation  : False
```

Comment:
I suggest placing this flag right under `use_tls`, to highlight the future potential for RA-TLS

Reply:
done